agent/commands/setup.go

```diff
@@ -46,9 +45,11 @@ func Setup() {
 
 	l := logrus.WithField("component", "setup")
 	cfg, configFilepath, err := config.Get(l)
-	var e *config.ConfigFileDoesNotExistError
```

Comment:
It seems that this bug can be fixed just by removing `*` in this line.

Reply:
@artemgavrilov I think `!` before `errors.As` confuses people and that's why we are having this discussion :)

Reply:
@artemgavrilov got it, pushed changes